library-best-practices/glossary.md

```diff
@@ -1,12 +1,20 @@
 Java Dependency Glossary
 ------------------------
 
-- **Class path**: an ordered list of jar files, zip files, or directories, each of which
+- **Class path**: an ordered list of jar files, zip files, and directories, each of which
   contains Java class files.
-  A class loader searches for a class file by its name through the path entries in a class path.
-  When there are two or more path entries (for example, jar files) that contain class files with
-  the same name in a class path, the class file in the first path entry in the class path
-  is available for use, and the other class files with the same name are unavailable.
+  A [class loader](https://docs.oracle.com/javase/7/docs/api/java/lang/ClassLoader.html)
+  typically transforms the fully package qualified name of a Java class
```

Comment:
"fully qualified" -> "fully-qualified"

Reply:
No hyphen in this Oracle doc

>  the fully qualified name of the Rectangle class in the java.awt package is java.awt.Rectangle.
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html